framework/src/interfacekernels/InterfaceKernel.C

```diff
@@ -87,10 +223,38 @@ InterfaceKernel::computeElemNeighJacobian(Moose::DGJacobianType type)
                                                         _neighbor_var.number(),
                                                         _neighbor_var.number());
 
+  _local_kxx.resize(Kxx.m(), Kxx.n());
+  _local_kxx.zero();
+
   for (_qp = 0; _qp < _qrule->n_points(); _qp++)
     for (_i = 0; _i < test_space.size(); _i++)
       for (_j = 0; _j < loc_phi.size(); _j++)
-        Kxx(_i, _j) += _JxW[_qp] * _coord[_qp] * computeQpJacobian(type);
+        _local_kxx(_i, _j) += _JxW[_qp] * _coord[_qp] * computeQpJacobian(type);
+
+  Kxx += _local_kxx;
+
+  if (_has_master_jacobians_saved_in && type == Moose::ElementElement)
+  {
+    unsigned int rows = _local_kxx.m();
```

Comment:
Are you sure `m()` returns `unsigned int`? Good place for `auto` (catching the right return type every time).